src/main/java/com/example/bandservice/controller/BandController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.bandservice.controller;
+
+import com.example.bandservice.model.Band;
+import com.example.bandservice.service.BandService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/bands")
```

Comment:
It is a good practice to add "/api/v1", for example, to a basepath

Reply:
Ok, ty